ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimMapper.java

```diff
@@ -94,11 +103,13 @@ public void to(Claim claim, CCDCase.CCDCaseBuilder builder) {
 
     public void from(CCDCase ccdCase, Claim.ClaimBuilder claimBuilder) {
         Objects.requireNonNull(ccdCase, "ccdClaim must not be null");
```

Comment:
Suggestion... Change the failure message to "ccdCase must be null"

Reply:
done